google-cloud-billing/.repo-metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "secret_manager",
```

Comment:
Replace Secret Manager naming everywhere in this file.

Reply:
Whups, missed those. Thanks!